rules/false_positives/pulumi.yara

```diff
@@ -0,0 +1,12 @@
+rule pulumi_binary : override {
+  meta:
+    description = "pulumi"
+    malware_shellcode_hash = "high"
```

Comment:
hmm - that's two places I see `malware_shellcode_hash`: after reviewing the JP CERT rule, I recommend disabling it entirely. It's not helpful at all: at best, it's medium priority.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).